Sources/SwiftGenKit/Parsers/Fonts/FontsParser.swift

```diff
@@ -32,7 +41,13 @@ public enum Fonts {
         return
       }
 
-      let fonts = CTFont.parse(file: path, relativeTo: parent)
+      let getCodePoints: Bool = self.options[Option.codePoints] == "true"
```

Comment:
We need to check how we do string-to-boolean for other config flags. I don't think we have many Boolean Parser Options that we can use as reference but we do have some template parameters that are Boolean (e.g. `forceNameSpaces` for the Assets templates); if there is similar cases, better ensure we do the conversion the same way (`== "true"` vs `!= "false"`, case-insensitivity, etc)

Reply:
Might actually be a nice addition to add to `ParserOptionValues` by overloading the subscript si that if the return type is inverted to `Bool`, that `ParserOptionValues` is the one doing the string-to-Bool conversion. That would allow us to ensure that all future Boolean parser options are treated the same way.